python/create_service_diagnostics_bundle.sh

```diff
@@ -42,30 +53,21 @@ else
   readonly CONTAINER_DCOS_SERVICE_DIAGNOSTIC_VOLUME_MOUNT=
 fi
 
-readonly CONTAINER_SCRIPT_PATH="${CONTAINER_DCOS_SERVICE_DIAGNOSTICS_DIRECTORY}/${SCRIPT_NAME}"
-
-function version () {
-  echo "${VERSION}"
-}
-
-if [ "${#}" -eq 1 ] && [[ "${1}" =~ ^(--version|-version|version|--v|-v)$ ]]; then
-  version
-  exit 0
-fi
-
-readonly REQUIREMENTS=('docker' 'dcos')
+# ###
+# 3. section: Define functions
+# ###
+function run_container() {
+  # run python diagnostic script as a processes in isolated container
 
-for requirement in ${REQUIREMENTS}; do
-  if ! [[ -x $(command -v "${requirement}") ]]; then
-    echo "You need to install '${requirement}' to run this script"
-    exit 1
-  fi
-done
+  # Docker process configuration
```

Comment:
This comment looks a bit confusing to me. "Docker process"? IMO let's remove it.

Reply:
removed